tests/Feature/JwtAuthorizationTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+
+namespace Marketredesign\MrdAuth0Laravel\Tests\Feature;
+
+use Closure;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use Illuminate\Testing\TestResponse;
+use Marketredesign\MrdAuth0Laravel\Tests\TestCase;
+
+class JwtAuthorizationTest extends TestCase
+{
+    private const ROUTE_URI = 'test';
+
+    /**
+     * Define environment setup.
+     *
+     * @param  Application  $app
+     * @return void
+     */
+    protected function getEnvironmentSetUp($app)
```

Comment:
```suggestion
    protected function setEnvironmentSetUp($app)
```

Reply:
This function doesn't return anything. Do you mean set i.o. get? Or perhaps just setupEnvironment

Reply:
This overwrites a function in the parent (Orchestra\Testbench\TestCase), so no way to change the name.